Makefile

```diff
@@ -86,9 +89,20 @@ help: ## Display this help.
 
 ##@ Development
 
+CONTROLLER_GEN = $(LOCAL_BIN)/controller-gen
+KUSTOMIZE = $(LOCAL_BIN)/kustomize
+ENVTEST = $(LOCAL_BIN)/setup-envtest
+GO_BINDATA = $(LOCAL_BIN)/go-bindata
+KUSTOMIZE_VERSION ?= v5.0.1
+OPM_VERSION ?= v1.27.0
+GO_BINDATA_VERSION ?= v3.1.2+incompatible
+BATS_VERSION ?= 1.2.1
+OLM_VERSION ?= v0.18.2
+KUBERNETES_VERSION ?= v1.26.4
+
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases output:rbac:dir=config/rbac/base
```

Comment:
why base removed?

Reply:
The approach before this PR was that if the distribution was OpenShift, then additional OpenShift specific permissions were added in the "overlays" directory. I decided it was better to always have the OpenShift permissions because they do not change anything on non-OpenShift clusters and then we can use the same build on vanilla Kubernetes and OpenShift.

So I removed the "overlays" directory so the separate "base" directory wasn't needed anymore.